Compilation error in NUnitPortableDriverTests.cs:

I discovered a compilation error in `master`.  It appears that it was assumed `System.Xml.Linq.XContainer.Elements(System.Xml.Linq.XName)` returned a type with a `Count` property.  However, because it is a `System.Collections.Generic.IEnumerable<T>` it instead has access to the `System.Linq.Count<TSource>(this IEnumerable<TSource>)` extension.  Because the property was expected to be an `int` and instead is a method group, that caused ambiguity with `Assert.That`.  It is possible I am failing to build correctly, but I followed the instructions in BUILDING.md and the [documentation](https://github.com/nunit/dev/wiki/Packaging-NUnit-3.0).

From the compiler:

```
NUnitPortableDriverTests.cs(77,13): error CS0121: The call is ambiguous between the following methods or properties: 'NUnit.Framework.Assert.That<int>(NUnit.Framework.Constraints.ActualValueDelegate<int>, NUnit.Framework.Constraints.IResolveConstraint, string, params object[])' and 'NUnit.Framework.Assert.That(NUnit.Framework.TestDelegate, NUnit.Framework.Constraints.IResolveConstraint, string, params string[])' [C:\Source\git\nunit\src\NUnitEngine\Portable\nunit.portable.agent.tests\nunit.portable.agent.tests.csproj]
NUnitPortableDriverTests.cs(92,13): error CS0121: The call is ambiguous between the following methods or properties: 'NUnit.Framework.Assert.That<int>(NUnit.Framework.Constraints.ActualValueDelegate<int>, NUnit.Framework.Constraints.IResolveConstraint, string, params object[])' and 'NUnit.Framework.Assert.That(NUnit.Framework.TestDelegate, NUnit.Framework.Constraints.IResolveConstraint, string, params string[])' [C:\Source\git\nunit\src\NUnitEngine\Portable\nunit.portable.agent.tests\nunit.portable.agent.tests.csproj]
NUnitPortableDriverTests.cs(141,13): error CS0121: The call is ambiguous between the following methods or properties: 'NUnit.Framework.Assert.That<int>(NUnit.Framework.Constraints.ActualValueDelegate<int>, NUnit.Framework.Constraints.IResolveConstraint, string, params object[])' and 'NUnit.Framework.Assert.That(NUnit.Framework.TestDelegate, NUnit.Framework.Constraints.IResolveConstraint, string, params string[])' [C:\Source\git\nunit\src\NUnitEngine\Portable\nunit.portable.agent.tests\nunit.portable.agent.tests.csproj]
```

Code in question:

```csharp
Assert.That(result.Elements("test-suite").Count, Is.EqualTo(0), "Load result should not have child tests");
```

Fixed:

```csharp
Assert.That(result.Elements("test-suite").Count(), Is.EqualTo(0), "Load result should not have child tests");
```

--- a/src/NUnitEngine/Portable/nunit.portable.agent.tests/NUnitPortableDriverTests.cs
+++ b/src/NUnitEngine/Portable/nunit.portable.agent.tests/NUnitPortableDriverTests.cs
@@ -74,7 +74,7 @@ namespace NUnit.Engine.Tests
             Assert.That(result.GetAttribute("type"), Is.EqualTo("Assembly"));
             Assert.That(result.GetAttribute("runstate"), Is.EqualTo("Runnable"));
             Assert.That(result.GetAttribute("testcasecount"), Is.EqualTo(MockAssembly.Tests.ToString()));
-            Assert.That(result.Elements("test-suite").Count, Is.EqualTo(0), "Load result should not have child tests");
+            Assert.That(result.Elements("test-suite").Count(), Is.EqualTo(0), "Load result should not have child tests");
         }
         #endregion
 
@@ -89,7 +89,7 @@ namespace NUnit.Engine.Tests
             Assert.That(result.GetAttribute("type"), Is.EqualTo("Assembly"));
             Assert.That(result.GetAttribute("runstate"), Is.EqualTo("Runnable"));
             Assert.That(result.GetAttribute("testcasecount"), Is.EqualTo(MockAssembly.Tests.ToString()));
-            Assert.That(result.Elements("test-suite").Count, Is.GreaterThan(0), "Explore result should have child tests");
+            Assert.That(result.Elements("test-suite").Count(), Is.GreaterThan(0), "Explore result should have child tests");
         }
 
         [Test]
@@ -138,7 +138,7 @@ namespace NUnit.Engine.Tests
             Assert.That(result.GetAttribute("failed"), Is.EqualTo(MockAssembly.ErrorsAndFailures.ToString()));
             Assert.That(result.GetAttribute("skipped"), Is.EqualTo(MockAssembly.Skipped.ToString()));
             Assert.That(result.GetAttribute("inconclusive"), Is.EqualTo(MockAssembly.Inconclusive.ToString()));
-            Assert.That(result.Elements("test-suite").Count, Is.GreaterThan(0), "Explore result should have child tests");
+            Assert.That(result.Elements("test-suite").Count(), Is.GreaterThan(0), "Explore result should have child tests");
         }
 
         [Test]

